### DISCUSSION TITLE
Page A has a model, but page B does not. When I switch back and forth between the two pages, it crashes

### DISCUSSION THREAD

**Discussion by shenhaoguangDayDayUp** [google/model-viewer#3575] [Q&A]
### Description
hi, Page A has a model, but page B does not. When I switch back and forth between the two pages, it crashes
<img width="1215" alt="image" src="https://user-images.githubusercontent.com/17617821/176846159-69e68ff9-fd3a-43e3-a945-c22159bf2be0.png">

<img width="890" alt="image" src="https://user-images.githubusercontent.com/17617821/176847990-2bc1e04a-9a8c-4603-b5d2-a9cdf86f57aa.png">
<img width="706" alt="image" src="https://user-images.githubusercontent.com/17617821/176848423-ac411de6-0de9-43a0-8d52-ce0b85a09683.png">

#### Live Demo

https://beautiful-seahorse-02fb7b.netlify.app

### Version

- model-viewer: v1.12.0

### Browser Affected

- [x] Matemask App
### OS

- [x] iOS

### AR

- [ ] WebXR
- [ ] SceneViewer
- [ ] QuickLook

**Comment by elalish** [comment]
I'm not sure how to reproduce from your live demo link. Can you list some specific steps? Also, can you reproduce using any of our dev pages? Or on any other browsers/devices? I wonder if something caused the gl context to be lost, though we ought to be able to recover from that too...

**Comment by shenhaoguangDayDayUp** [comment]
> I'm not sure how to reproduce from your live demo link. Can you list some specific steps? Also, can you reproduce using any of our dev pages? Or on any other browsers/devices? I wonder if something caused the gl context to be lost, though we ought to be able to recover from that too...

thanks for your  reply, this is  codepan link https://codesandbox.io/s/vue-reactivity-forked-v83go8?file=/src/components/Tab.vue


You can use this link(https://v83go8.csb.app/#/model) on your iphone , switch back and forth between two tabs multiple times. an error will be reported

![998501656736074_ pic](https://user-images.githubusercontent.com/17617821/176986481-1647187b-8ed6-4856-af37-a8c4d43e2e45.jpg)

**Comment by shenhaoguangDayDayUp** [comment]
> 我不确定如何从您的现场演示链接中重现。你能列出一些具体的步骤吗？另外，您可以使用我们的任何开发页面进行复制吗？或者在任何其他浏览器/设备上？我想知道是否有什么东西导致了 gl 上下文丢失，尽管我们也应该能够从中恢复......

Hello, Did you find the problem?

**Comment by elalish** [comment]
Sorry, no. On my old iPhone SE your site is working just fine (I toggled back and forth at least a dozen times). It could be a Safari WebGL bug; they've had some issues with their ImageBitmap implementation. Are you on the most recent version of iOS and Safari?

**Reply by shenhaoguangDayDayUp** [reply]
I am using iphone 12 and the system version is 14.6,Can this problem be solved or is there any solution?

**Reply by shenhaoguangDayDayUp** [reply]
> I am using iphone 12 and the system version is 14.6,Can this problem be solved or is there any solution?

https://user-images.githubusercontent.com/17617821/177891074-c4628f42-347e-4aec-85e5-796f1dbf5d3b.mp4

**Reply by elalish** [reply]
Try upgrading to iOS 15? Unfortunately there's often not much we can do about WebGL browser bugs.

**Reply by shenhaoguangDayDayUp** [reply]
> Try upgrading to iOS 15? Unfortunately there's often not much we can do about WebGL browser bugs.

OK, Thank you very much. I will try.
